common/app/views/fragments/commercial/adSlot.scala.html

```diff
@@ -25,9 +25,9 @@
     }
     >@placeholderContent
 
-    @if(!request.isAmp) {
-        @Commercial.getBlockthroughElementUid(name).map { uid =>
-            <span class="bt-uid-tg" uid="@uid" style="display: none !important"></span>
-        }
-    }
 </div>
+@if(!request.isAmp) {
+    @Commercial.getBlockthroughElementUid(name).map { uid =>
+<span class="bt-uid-tg" uid="@uid" style="display: none !important"></span>
+    }
+}
```

Comment:
Is this a deliberate change?

Reply:
Yes, the 'Blockthrough' span must be outside the main adslot for blockthrought to work. See https://trello.com/c/M2rgtH0O/158-amend-placement-of-blockthrough-placement-to-existing-ad-slots

Reply:
I see.  So it's moved outside the div.  But indenting looks a bit strange.

Reply:
I wanted it to be at the same indentation level as the above div, so it looks neat in the page source.

Reply:
OK 👍